cells/lib/ops/reviseOCI.nix

```diff
@@ -0,0 +1,63 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs;
+  l = nixpkgs.lib // builtins;
+in
+  /*
+  Utility function to allow for building containers in a mono-repo style environment where
+  the source code is contained in the same repository as the std code, specifically so that
```

Comment:
As above: unecessary restriction?

Reply:
Unless you can give me an example I'm not thinking of, I don't think so

Reply:
It has two uses, I think both spelled out here. What other usecases could you imagine?